### PR TITLE
Add missing string in index to translation

### PR DESCRIPTION
**Pull request by YUCLing**

### CODE REVIEW

**Comment by hbjydev** [comment]
#### 📝Changed routes:


| [**/**](https://rockylinux-eaysvmxh0-resforg.vercel.app/) |
|:-:|
|<a href="https://rockylinux-eaysvmxh0-resforg.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/4yWN01uidN-h1z5I3c-aaYhhLpCkEm4yVJiFi0KJb2lJ9Z6eqFn0eFeVnF8OoxMI8g5qcIx6RzgfymvuOqsA6w.png">(view full size)</a> |







Commit 35a0cbd65654326921c7ffc3e7fa3256eb53a185 (https://rockylinux-eaysvmxh0-resforg.vercel.app).

File: src/components/footer/index.jsx [hunk 1]
```diff
@@ -224,23 +224,23 @@ const Footer = ({ pageContext: { locale: language } }) => {
               to="/community-charter/"
               className="text-sm text-gray-500 hover:text-gray-400"
             >
-              Community Charter
+              {translate('links.community-harter')}
```
Comment (hbjydev) [inline src/components/footer/index.jsx]:
```suggestion
              {translate('links.community-charter')}
```

**Review by hbjydev — CHANGES_REQUESTED** [review]

**Review by hbjydev — CHANGES_REQUESTED** [review]
If we're going to do this, I think we need to add an i18n-next provider now. It will let us specify the _fallback_ language. Currently, on other languages, just the translation string is shown.

**Review by YUCLing — COMMENTED** [review]
All files seem to be ok now

**Comment by hbjydev** [comment]
We need your language changes to be peer reviewed but then I'll give it a merge :)

**Pull request closed without merging**